snap/snapcraft.yaml

```diff
@@ -777,11 +820,12 @@ parts:
   device-virtual-go:
     source: https://github.com/edgexfoundry/device-virtual-go.git
     source-depth: 1
-    source-tag: master
+    # TODO: ideally this should be changed to v2.0.0
+    source-tag: v2.0.0
     plugin: make
     after: [go-build-helper]
     override-build: |
-      echo "2.0.0-dev" > ./VERSION
+      echo "2.0.0" > ./VERSION
```

Comment:
nit: ideally this should grab the git tag, but OK for now...

Reply:
let's note it for a later update. I suppose we're tagging it against  v2.0.0 anyway so it's going to remain 2.0.0 unless we change that.